app/views/images/show.html.erb

```diff
@@ -1,5 +1,5 @@
 <% content_for :head_script do %>
-<%= javascript_include_tag 'lightbox' %>
+<% javascript_include_tag 'openseadragon' %>
```

Comment:
I've just learned something new.  Most browsers now support `async` and `defer` attributes for the `script` element, per http://stackoverflow.com/questions/436411/where-is-the-best-place-to-put-script-tags-in-html-markup, so I'm looking into whether I can do that with `javascript_include_tag`.